librz/asm/arch/hexagon/hexagon.h

```diff
@@ -135,6 +138,7 @@ typedef struct {
 	RzList *const_ext_l; // Constant extender values.
 	RzAsm rz_asm; // Copy of RzAsm struct. Holds certain flags of interesed for disassembly formatting.
```

Comment:
Same - please convert other comments to this struct members' to Doxygen.

Reply:
I do this in another PR. This way all comments can be updated at once.